### PR TITLE
add backlog documentation

### PR DESCRIPTION
**Pull request by AlexExcellence**

### CODE REVIEW

File: docs/backlog/backlog.rst [hunk 1]
```diff
@@ -1,4 +1,52 @@
 Product Backlog
 ===========================================
+* Create App Icon/Name
```
Comment (pal03377) [inline docs/backlog/backlog.rst]: Some introductory sentence would be nice :)

File: docs/backlog/backlog.rst [hunk 2]
```diff
@@ -1,4 +1,52 @@
...
+      * Implement a Functionality to edit/delete inline/general Feedbacks
+      * Tapping on inline Feedbacks results in showing the related Code Segment in the Code Editor
+  - Implement a Functionality to save the current Progress of an Assessment
+  - Implement a Functionality to submit the Assessment
+  - Implement a Functionality to dismiss the Assessment
+* Setup Server for ML Functionality
+* Notify ML Server when Submitting Assessments
+* Store Function Blocks with related Feedback in Database
+* Implement Similarity Comparison for Function Blocks of Submissions with stored Blocks in Database
+* Connect App to ML Server
```
Comment (pal03377) [inline docs/backlog/backlog.rst]: These are only the implemented backlog items, right? The remaining ones will be added as well later?

**Review by pal03377 — CHANGES_REQUESTED** [review]
The capitalization is a bit inconsistent. I would suggest either using Title Case for everything or writing the items in lowercase.

File: docs/backlog/backlog.rst (hunk 2 above)
Comment (AlexExcellence) [reply docs/backlog/backlog.rst]: The remaining ones will be converted to Github Issues, which is why we won't write them down.

File: docs/backlog/backlog.rst (hunk 1 above)
Comment (botrufus) [reply docs/backlog/backlog.rst]: Yes, I also think its a bit of a blunt chapter without any introduction

File: docs/backlog/backlog.rst [hunk 3]
```diff
@@ -1,4 +1,52 @@
...
+  
+  - use Bearer Token / Cookie based Authentication based on Artemis Version
+* Implement a Functionality to view and switch between the Tutors Courses
+* Implement a View that shows all Exercises for a Course and allows selecting them
 
-by Alexander Lange 
\ No newline at end of file
+  - Show a Timeline indicating Release-, Submission- and Assesment (Due) Dates
+* Implement a View that shows Assessment Statistics for a specific Exercise
+* Implement a Functionality to search for a specific Students Submission
```
Comment (crustypluto19) [inline docs/backlog/backlog.rst]: "Implement a Functionality" sound weird IMO, I think it should just be "Implement functionality to..."

**Review by crustypluto19 — COMMENTED** [review]
As Paul mentioned, capitalization is inconsistent, but good job nonetheless!

**Review by botrufus — CHANGES_REQUESTED** [review]
Fine

**Review by botrufus — APPROVED** [review]
looks good

**Review by pal03377 — APPROVED** [review]
Well done!

**Review by crustypluto19 — APPROVED** [review]

**Pull request merged**